substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1982,8 +1982,9 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// Stake funds with a pool. The amount to bond is transferred from the member to the
-		/// pools account and immediately increases the pools bond.
+		/// Stake funds with a pool. The amount to bond is delegated (or transferred based on
+		/// [`adapter::StakeStrategyType`]) from the member to the pool account and immediately
```

Comment:
These are the docs that will be shown in e.g. TS and PJS apps, perhaps it can be informative.